ovos_PHAL_plugin_homeassistant/logic/socketclient.py

```diff
@@ -46,7 +46,7 @@ async def authenticate(self):
     async def _connect(self):
         try:
             uri = f"{self.url}/api/websocket"
-            self.websocket = await websockets.connect(uri)
+            self.websocket = await websockets.connect(uri=uri, close_timeout=5, open_timeout=5)
```

Comment:
Any reason to make `open_timeout` configurable? 5s seems like plenty but I'm not sure if that includes authentication/handshaking that could take longer for some reason

Reply:
It seems like a lot of extra work for minimal gain, but I'm open to the idea